package.json

```diff
@@ -29,5 +30,14 @@
     "which": "^1.1.1"
   },
   "homepage": "https://github.com/jspm/npm",
-  "devDependencies": {}
+  "devDependencies": {
+    "istanbul": "0.3.21",
+    "lodash": "3.10.1",
+    "magicpen-media": "1.5.1",
+    "messy": "6.6.2",
+    "mocha": "2.3.3",
+    "unexpected": "9.16.0",
+    "unexpected-messy": "4.8.1",
```

Comment:
Here I'm addressing this comment, as well as https://github.com/jspm/npm/pull/97#discussion_r40607300

The devDependency on lodash is removed now, as mentioned in another comment.

istanbul is added for generating code coverage reports. I think it is a valuable tool to help find areas of the code that you haven't hit with the unit tests yet. mocha is the test runner.

unexpected is the assertion framework, and due to it being build with a slim core and a [bunch of plugins](http://unexpected.js.org/plugins/), and due to how npm works with peerDependencies, it has the unfortunate side effect of installing all plugins that the plugins that you rely on rely on... 

- We use unexpected with the plugin [unexpected-mitm](https://github.com/unexpectedjs/unexpected-mitm/) which wraps a module called man-in-the-middle, which is used for mocking out HTTP connections on the socket level.

- unexpected-mitm relies on another unexpected plugin called [unexpected-messy](http://unexpected.js.org/unexpected-messy/) which provides types and assertions for HTTP conversations. 

- unexpected-messy wraps a library called [messy](https://github.com/papandreou/messy) which provides data models for email and http conversations. It also has a dependency (which really should be an optional dependency, but we have no way of expressing that, other than a peerDependency) on magicpen-media, which supports showing images in test output, if you are running those tests in an environment that supports it (e.g. a browser).

This problem will be solved with npm@3 as that will no longer cause peerDependencies to automatically be installed, and it will automatically flatten the dependency tree, so that we won't get more instances of the same plugin (which is one of the major pain points, that caused us to accept this situation with excessive dependencies). Then you'll have to manually install the optional plugins, when you want to use them, which makes a bit more sense.

So... to wrap up: All of the dependencies are used by unexpected - so none of them can be removed. The situation will be better once npm@3 gets out and we can assume that everyone uses that. And they are development dependencies, so they will not be installed when you install this module as a user of jspm. So I think that it is acceptable. :-)